app/src/hvr/java/com/igalia/wolvic/PlatformActivity.java

```diff
@@ -246,8 +246,9 @@ private void initializeAGConnect() {
             try {
                 SpeechRecognizer speechRecognizer = SpeechServices.getInstance(this, speechService);
                 ((VRBrowserApplication) getApplicationContext()).setSpeechRecognizer(speechRecognizer);
-            } catch (ReflectiveOperationException e) {
-                e.printStackTrace();
+            } catch (Exception e) {
+                Log.e(LOGTAG, "Exception creating the speech recognizer: " + e);
```

Comment:
Looks like LOGTAG is not defined here.

Reply:
Sorry about that, it should be fixed now.